crypto/vault/vault.go

```diff
@@ -1,12 +1,12 @@
 package vault
 
 import (
+	"github.com/drand/drand/crypto"
 	"sync"
 
-	"github.com/drand/drand/chain"
-	"github.com/drand/drand/crypto"
-	"github.com/drand/drand/key"
-	"github.com/drand/drand/log"
+	"github.com/drand/drand/common/chain"
+	key2 "github.com/drand/drand/common/key"
```

Comment:
why a named import? There's no other `key` here, is there?

Reply:
Fixed